README.md

```diff
@@ -173,7 +173,11 @@ To enable telescope integration:
 ```lua
 require('telescope').load_extension('projects')
 ```
-
+### Telescope Projects Picker
```

Comment:
```suggestion

#### Telescope Projects Picker
```

This should be a level 4 header, also make sure to add some spacing before the header!